pkg/reconciler/autoscaling/kpa/scaler.go

```diff
@@ -182,15 +182,7 @@ func (ks *scaler) handleScaleToZero(ctx context.Context, pa *pav1alpha1.PodAutos
 	now := time.Now()
 	logger := logging.FromContext(ctx)
 	switch {
-	case pa.Status.IsActivating(): // Active=Unknown
-		// If we are stuck activating for longer than our progress deadline, presume we cannot succeed and scale to 0.
-		if pa.Status.CanFailActivation(now, activationTimeout) {
-			logger.Info("Activation has timed out after ", activationTimeout)
-			return desiredScale, true
-		}
-		ks.enqueueCB(pa, activationTimeout)
-		return scaleUnknown, false
-	case pa.Status.IsReady(): // Active=True
+	case pa.Status.IsActive(): // Active=True
```

Comment:
Same here?

Reply:
I think we want to use the Active condition here for symmetry with the other switch cases (whereas readiness will also include having observed the latest generation)